### DISCUSSION TITLE
Atom feed questions, validation

### DISCUSSION THREAD

**Discussion by JVimes** [welpo/tabi#391] [General]
I noticed the Atom feed has some [warnigns](https://validator.w3.org/feed/check.cgi?url=https%3A%2F%2Fwelpo.github.io%2Ftabi%2Fatom.xml) from W3's validator: "unknown namespace", "undefined element", and "str generally associated with [other URL]".

Are these a concern, or for practical reasons?  
Do article images usually show up in feed readers?

Just trying to make sure my feed is set up right before it gets cached by reader services. 😝

**Comment by welpo** [comment]
I'm pretty sure none of these warnings affect the functionality of the feed.

That said, I'm addressing them with #393. Some are unavoidable:
- Same-document reference: happens when feed is for main site default lang or not for a section. Necessary to add the "Visit website →" part
- Use of unknown namespace: I don't know if this can be fixed. I need to put the metadata for the XSL transformation somewhere.

Thanks for the report!

---

> Do article images usually show up in feed readers?

They should, if you set `full_content_in_feed = true` ([docs](https://welpo.github.io/tabi/blog/mastering-tabi-settings/#full-content-in-feed)).

**Reply by JVimes** [reply]
Thanks! I'm sure it's fine. I like that the feed XML has a "page".
